proxy/stats_get_prediction.go

```diff
@@ -66,6 +68,20 @@ func (rw *CachingResponseWriter) Write(data []byte) (int, error) {
 	return rw.w.Write(data)
 }
 
+func (s *StatsGetPrediction) proxyRequest(r *http.Request) (*http.Response, error) {
```

Comment:
Isn't this the same function in proxy.go? Seems better to just maintain a reference to the `StriveAPIProxy` struct in the `StatsGetPrediction` so you don't need to dupe code.